packages/vite/package.json

```diff
@@ -60,7 +60,7 @@
     "@rollup/plugin-dynamic-import-vars": "^1.1.1",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^11.0.1",
-    "@rollup/plugin-typescript": "^8.0.0",
+    "@rollup/plugin-typescript": "^8.2.1",
```

Comment:
Is the version bumpping required?

Reply:
I saw there is a problem with the previous version (see https://stackoverflow.com/questions/63218218/rollup-is-not-generating-typescript-sourcemap), but seems like it has been resolved in the latest version.